docs/content/en/docs/install/_index.md

```diff
@@ -19,46 +19,71 @@ Your use of this software is subject to the [Google Privacy Policy](https://poli
 
 {{% tabs %}}
 {{% tab "LINUX" %}}
-The latest **stable** binary can be found here:
+The latest **stable** binaries can be found here:
 
 https://storage.googleapis.com/skaffold/releases/latest/skaffold-linux-amd64
+https://storage.googleapis.com/skaffold/releases/latest/skaffold-linux-arm64
 
-Simply download it and add it to your `PATH`. Or, copy+paste this command in your terminal:
+Simply download the appropriate binary and add it to your `PATH`. Or, copy+paste this command in your terminal:
 
 ```bash
+# For Linux AMD64
 curl -Lo skaffold https://storage.googleapis.com/skaffold/releases/latest/skaffold-linux-amd64 && \
 sudo install skaffold /usr/local/bin/
+
+# For Linux ARM64
+curl -Lo skaffold https://storage.googleapis.com/skaffold/releases/latest/skaffold-linux-arm && \
+sudo install skaffold /usr/local/bin/
+```
 ```
```

Comment:
Looks like there's a problem on the install page, I imagine from not putting a line in between the start of the other monospace block http://34.94.250.235:1313/docs/install/

Reply:
Oh jeepers.  That'll teach me for making a simple change without verifying.